src/mk/tools/just.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+import os
+import re
+from pathlib import Path
+
+from mk.exec import run_or_fail
+from mk.tools import Action, Tool
+
+
+class JustTool(Tool):
+    name = "just"
+
+    def __init__(self) -> None:
+        super().__init__(self)
+        self.justfile: str | None = None
+
+    def run(self, action: Action | None = None) -> None:
+        cmd = ["just"]
+        if action:
+            cmd.append(action.name)
+        run_or_fail(cmd, tee=True)
+
+    def is_present(self, path: Path) -> bool:
+        for name in ["Justfile", "justfile", "JUSTFILE", ".justfile"]:
+            justfile = os.path.join(path, name)
+            if os.path.isfile(justfile):
```

Comment:
```suggestion
            justfile = path / name
            if justfile.is_file():
```

Reply:
This file was written as similarly to `make.py` as possible. If this is needed, I think another PR could convert more of the old-style path things at once?